default/src/components/signer.js

```diff
@@ -120,6 +128,139 @@ const GetTxParams = () => {
   );
 };
 
+const SendASA = () => {
+  let asaData = YAMLData.default.default.asa;
+  let deployedASAName;
+  let deployedASAId;
+  let asaId;
+  for (const [key, value] of Object.entries(asaData)) {
+    deployedASAName = key;
+    deployedASAId = value.assetIndex;
+  }
+
+  const [result, setResult] = useState('');
+
+  const action1 = useCallback(async () => {
+    asaId = deployedASAId;
+  });
+
+  const action2 = useCallback(async () => {
+    asaId = prompt('Please enter ASA ID');
+  });
+
+  const transfer = useCallback(async () => {
+    try {
+      let receiver = document.getElementById('recvAddr').value;
+      if (receiver === '') {
+        return "Please enter receiver's address.";
+      }
+      let sender = document.getElementById('sndrAddr').value;
+      if (sender === '') {
+        return "Please enter sender's address.";
+      }
+      let amount = document.getElementById('amount').value;
+      if (amount === '') {
+        return 'Please enter amount.';
+      }
+      if (!Number(asaId)) {
+        return 'Entered ASA ID is not a number.';
+      }
+      if (!Number(amount)) {
+        return 'Entered amount is not a number.';
```

Comment:
amount, asaId can be `bigint` as well (to support upto *uint64*)

Reply:
Bigint amount was rejected by `algosdk`.

Reply:
That doesn't sound right. Are you sure you're using the latest or same version of `algosdk` like we do in algo-builder?

Reply:
> Bigint amount was rejected by `algosdk`.

It was not rejected - they didn't implement it yet. so maybe we can make a TODO and udpate the type once SDK will support it.

Reply:
Sure.